source/sonata_tech.rst

```diff
@@ -478,7 +478,7 @@ Connection type is ``neuromodulatory``. Used for neuromodulatory projections bet
 Fields for "point neuron connectivity"
```

Comment:
```suggestion
Fields for "point neuron Tsodyks Markram model connectivity"
```
perhaps (so it shows up in the index)?

Reply:
Changes are applied. Can we merge this PR ?